playback/src/player.rs

```diff
@@ -43,6 +44,14 @@ enum PlayerCommand {
     Seek(u32),
 }
 
+#[derive(Debug)]
+struct NormalisationConfig {
```

Comment:
This shouldn't be called `Config`. Something like `Data` or `Values` or `Parameters` would be more appropriate.